README.md

```diff
@@ -106,6 +106,20 @@ We can then use the `count` variable to display its current value in the
 // => <button>0</button>
 ```
 
+## Typing State 
+
+When initially setting state, TypeScript can infer the type to use based on
+the initial value. However, if it's desired to be more explicit about the type, 
+we can do so as well using the generic syntax `<>`. For example with our `count`
+state, we could write: 
+
+`const [count, setCount] = useState<number>(0);` 
+
+It may be tempting to type within the descrutctured syntax, but just as we are 
+unable to type within destructured objects, we cannot do so within destructured 
```

Comment:
Recommended rewording:

"You may recall from an earlier lesson that typing is not allowed within destructured objects; the same applies to destructured arrays as well. Typing must instead be done using the generic syntax, which will handle ..."